kubetest2/internal/deployers/eksapi/templates/unmanaged-nodegroup-efa.yaml

```diff
@@ -201,6 +205,9 @@ Resources:
               DeleteOnTermination: true
               VolumeSize: !Ref NodeDiskSize
               VolumeType: gp2
+        CapacityReservationSpecification: 
+          CapacityReservationTarget:
+            CapacityReservationId: !Ref CapacityReservationId
```

Comment:
same question here, what happens if we don't have a capacityReservationId?

Reply:
I updated the template to not specify the target when it is not set.